### PR TITLE
Org avatar fix

### PR DESCRIPTION
**Pull request by mcbouslog**
Staging branch URLs: 
with avatar - https://org-avatar-fix.pfe-preview.zooniverse.org/organizations/markb-panoptes/nfnorgtest
without avatar - https://org-avatar-fix.pfe-preview.zooniverse.org/organizations/markb-panoptes/noavnoback

Fixes - https://www.zooniverse.org/organizations/meredithspalmer/snapshot-safari.
Staged - https://org-avatar-fix.pfe-preview.zooniverse.org/organizations/meredithspalmer/snapshot-safari?env=production.

- adds `alt` text
- consolidates to one class for org avatar
- adds ZooniverseLogo as placeholder 
- refactors styling, improves mobile styling

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [x] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

**Comment by eatyourgreens** [comment]
On my phone, the organisation name and byline is ranged right, off the screen. Master does that too, but I wasn't sure if the style changes here were supposed to fix this bug.
Snapshot organisation screenshot, showing the avatar left-aligned and title ranged right off the screen.

File: css/organization-page.styl [hunk 1]
```diff
@@ -54,6 +58,9 @@
   min-height: 400px
   position: relative
 
+  @media (max-width: 900px)
+    min-height: 500px
```
Comment (mcbouslog) [inline css/organization-page.styl]: not thrilled with this solution, fits all of Snapshot Safari's hero elements, but might not fit all of a different org's. unsure of alternate solution.

**Comment by mcbouslog** [comment]
ExternalLinksBlock's `min-width: 400px` I think is causing:
screen shot 2018-04-25 at 2 42 04 pm

an issue on devices that are less than 400px (iPhoneX/6/7/8). I think beyond the scope of this PR, but wanted to note while thinking about.

**Comment by eatyourgreens** [comment]
Looks good on my 6plus. I’d say merge this and come back to address the 400px width in future PRs.

**Pull request merged**